tracee-ebpf/tracee/consts.go

```diff
@@ -903,4 +907,6 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
 	SecuritySocketAcceptEventID:  {{Type: "int", Name: "sockfd"}, {Type: "struct sockaddr*", Name: "local_addr"}},
 	SecuritySocketBindEventID:    {{Type: "int", Name: "sockfd"}, {Type: "struct sockaddr*", Name: "local_addr"}},
 	SecuritySbMountEventID:       {{Type: "const char*", Name: "dev_name"}, {Type: "const char*", Name: "path"}, {Type: "const char*", Name: "type"}, {Type: "unsigned long", Name: "flags"}},
+	SecurityBPFID:                {{Type: "int", Name: "map_cmd"}, {Type: "const char*", Name: "map_name"}},
+	SecurityBPFMapID:             {{Type: "const char*", Name: "map_name"}},
```

Comment:
I think that we should also have an argument for the map id (which is also available in the bpf_map struct) to uniquely identify the map, WDYT?

Reply:
Definitely!! Much better and clean. 

```
$ sudo bpftool map list 
351: hash  name args_map  flags 0x0
	key 8B  value 48B  max_entries 10240  memlock 1331200B
352: hash  name bin_args_map  flags 0x0
	key 8B  value 56B  max_entries 10240  memlock 1413120B
353: percpu_array  name bufs  flags 0x0
	key 4B  value 32768B  max_entries 3  memlock 2363392B
354: percpu_array  name bufs_off  flags 0x0
	key 4B  value 4B  max_entries 3  memlock 4096B
355: hash  name chosen_events_m  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
356: hash  name comm_filter  flags 0x0
	key 16B  value 4B  max_entries 10240  memlock 1003520B
357: hash  name config_map  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
358: perf_event_array  name events  flags 0x0
	key 4B  value 4B  max_entries 1024  memlock 12288B
359: array  name file_filter  flags 0x0
	key 4B  value 64B  max_entries 3  memlock 4096B
360: perf_event_array  name file_writes  flags 0x0
	key 4B  value 4B  max_entries 1024  memlock 12288B
361: hash  name inequality_filt  flags 0x0
	key 4B  value 8B  max_entries 10240  memlock 921600B
362: hash  name mnt_ns_filter  flags 0x0
	key 8B  value 4B  max_entries 10240  memlock 921600B
363: hash  name new_pidns_map  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
364: hash  name new_pids_map  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
365: hash  name params_names_ma  flags 0x0
	key 4B  value 8B  max_entries 10240  memlock 921600B
366: hash  name params_types_ma  flags 0x0
	key 4B  value 8B  max_entries 10240  memlock 921600B
367: hash  name pid_filter  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
368: hash  name pid_ns_filter  flags 0x0
	key 8B  value 4B  max_entries 10240  memlock 921600B
369: prog_array  name prog_array  flags 0x0
	key 4B  value 4B  max_entries 3  memlock 4096B
	owner_prog_type kprobe  owner jited
370: hash  name ret_map  flags 0x0
	key 8B  value 8B  max_entries 10240  memlock 921600B
371: hash  name sockfd_map  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
372: stack_trace  name stack_addresses  flags 0x0
	key 4B  value 160B  max_entries 1024  memlock 192512B
373: array  name string_store  flags 0x0
	key 4B  value 64B  max_entries 1  memlock 4096B
374: hash  name sys_32_to_64_ma  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
375: prog_array  name sys_enter_tails  flags 0x0
	key 4B  value 4B  max_entries 1022  memlock 12288B
	owner_prog_type raw_tracepoint  owner jited
376: prog_array  name sys_exit_tails  flags 0x0
	key 4B  value 4B  max_entries 1022  memlock 12288B
	owner_prog_type raw_tracepoint  owner jited
377: hash  name traced_pids_map  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
378: hash  name uid_filter  flags 0x0
	key 4B  value 4B  max_entries 10240  memlock 921600B
379: hash  name uts_ns_filter  flags 0x0
	key 16B  value 4B  max_entries 10240  memlock 1003520B
```

and

```
$ sudo ./dist/tracee-ebpf --debug --trace event=security_bpf_map --trace comm=bpftool
found bpf object file at: /home/rafaeldtinoco/work/sources/upstream/tracee/dist/tracee.bpf.5_8_0-43-generic.v0_5_1-24-g418d029.o
TIME(s)        UID    COMM             PID     TID     RET              EVENT                ARGS
247400.328408  0      bpftool          421620  421620  0                security_bpf_map     map_id: 351, map_name: args_map
247400.328722  0      bpftool          421620  421620  0                security_bpf_map     map_id: 352, map_name: bin_args_map
247400.328907  0      bpftool          421620  421620  0                security_bpf_map     map_id: 353, map_name: bufs
247400.329060  0      bpftool          421620  421620  0                security_bpf_map     map_id: 354, map_name: bufs_off
247400.329212  0      bpftool          421620  421620  0                security_bpf_map     map_id: 355, map_name: chosen_events_m
247400.329362  0      bpftool          421620  421620  0                security_bpf_map     map_id: 356, map_name: comm_filter
247400.329512  0      bpftool          421620  421620  0                security_bpf_map     map_id: 357, map_name: config_map
247400.329660  0      bpftool          421620  421620  0                security_bpf_map     map_id: 358, map_name: events
247400.329814  0      bpftool          421620  421620  0                security_bpf_map     map_id: 359, map_name: file_filter
247400.329965  0      bpftool          421620  421620  0                security_bpf_map     map_id: 360, map_name: file_writes
247400.330119  0      bpftool          421620  421620  0                security_bpf_map     map_id: 361, map_name: inequality_filt
247400.330269  0      bpftool          421620  421620  0                security_bpf_map     map_id: 362, map_name: mnt_ns_filter
247400.330419  0      bpftool          421620  421620  0                security_bpf_map     map_id: 363, map_name: new_pidns_map
247400.330569  0      bpftool          421620  421620  0                security_bpf_map     map_id: 364, map_name: new_pids_map
247400.330718  0      bpftool          421620  421620  0                security_bpf_map     map_id: 365, map_name: params_names_ma
247400.330867  0      bpftool          421620  421620  0                security_bpf_map     map_id: 366, map_name: params_types_ma
247400.331018  0      bpftool          421620  421620  0                security_bpf_map     map_id: 367, map_name: pid_filter
247400.331169  0      bpftool          421620  421620  0                security_bpf_map     map_id: 368, map_name: pid_ns_filter
247400.331320  0      bpftool          421620  421620  0                security_bpf_map     map_id: 369, map_name: prog_array
247400.331530  0      bpftool          421620  421620  0                security_bpf_map     map_id: 370, map_name: ret_map
247400.331680  0      bpftool          421620  421620  0                security_bpf_map     map_id: 371, map_name: sockfd_map
247400.331831  0      bpftool          421620  421620  0                security_bpf_map     map_id: 372, map_name: stack_addresses
247400.331981  0      bpftool          421620  421620  0                security_bpf_map     map_id: 373, map_name: string_store
247400.332152  0      bpftool          421620  421620  0                security_bpf_map     map_id: 374, map_name: sys_32_to_64_ma
247400.332315  0      bpftool          421620  421620  0                security_bpf_map     map_id: 375, map_name: sys_enter_tails
247400.332538  0      bpftool          421620  421620  0                security_bpf_map     map_id: 376, map_name: sys_exit_tails
247400.332761  0      bpftool          421620  421620  0                security_bpf_map     map_id: 377, map_name: traced_pids_map
247400.332956  0      bpftool          421620  421620  0                security_bpf_map     map_id: 378, map_name: uid_filter
247400.333119  0      bpftool          421620  421620  0                security_bpf_map     map_id: 379, map_name: uts_ns_filter
```